app/assets/javascripts/triggers.js

```diff
@@ -1,14 +1,17 @@
-var ready;
+var ready, addListeners;
 
-ready = function() {
-  $('#conditions').on('cocoon:after-insert', function(e, insertedItem) {
-    $("input.slider").slider();
-    $('.slider-range').on('slide', function(slider){
-      $(this).closest('.range-group').first().find('input.range-slider-min-value').val(slider.value[0]);
-      $(this).closest('.range-group').first().find('input.range-slider-max-value').val(slider.value[1]);
-    });
+addListeners = function() {
+  $("input.slider").slider();
+  $('.slider-range').on('slide', function(slider){
+    $(this).closest('.range-group').first().find('input.range-slider-min-value').val(slider.value[0]);
+    $(this).closest('.range-group').first().find('input.range-slider-max-value').val(slider.value[1]);
   });
 };
 
+ready = function() {
+  addListeners();
```

Comment:
For existing sensor conditions, the slider did not update the hidden input fields